packages/web/src/scss/components/Toast/_ToastBar.scss

```diff
@@ -0,0 +1,53 @@
+// 1. Restore pointer events that have been disabled by parent `Toast` component.
+
+@use '../../tools/dictionaries';
+@use '../../tools/typography';
+@use 'theme';
+
+.ToastBar {
+    max-width: theme.$bar-max-width;
+    padding: theme.$bar-padding;
+    border-radius: theme.$bar-border-radius;
+    box-shadow: theme.$bar-box-shadow;
+    pointer-events: initial; // 1.
+}
+
+.ToastBar--dismissible {
+    display: grid;
+    grid-template-columns: 1fr auto;
+    column-gap: theme.$gap;
+    align-items: start;
+}
+
+.ToastBar--dismissible > .ToastBar__content {
+    align-self: center;
+}
+
+.ToastBar__content:has(> svg:first-child) {
+    display: grid;
+    grid-template-columns: auto 1fr;
+    column-gap: theme.$bar-content-gap;
+}
+
+.ToastBar__message {
+    @include typography.generate(theme.$bar-typography);
+
+    display: flex;
+    flex-wrap: wrap;
+    gap: theme.$bar-message-gap-y theme.$bar-message-gap-x;
+}
+
+.ToastBar__message > :is(a, button):last-child {
+    font-weight: 400;
```

Comment:
There is not style for small links… I could use the "typography body small text" style, but I don't want it to interfere with possible link styles coming from the outside (from the developers). I'm still not sure how the action will be implemented, but I suppose it will be an external thing.

Reply:
Another approach would be to wrap the link into one more `<div>` with `typography.generate()` applied to it. But that's too many `<div>`s I think…

Reply:
ok, I think we will see how we implement it and how will the developers use it.